### DISCUSSION TITLE
Flathub Verification

### DISCUSSION THREAD

**Discussion by NullDott** [Orama-Interactive/Pixelorama#1055] [General]
This is more of a request for something than an idea, can we get Flatpak verification in Flathub? I think it would be good since the flatpak is an official build, and some distros like Linux Mint (the one I use) have been starting to hide unverified flatpaks by default (with a pretty dumb reasoning IMO).

It's just a little enhancement, and I think it would be good now that 1.0 is out, thanks for your attention.

**Comment by novhack** [comment]
The ID of the app com.orama_interactive.Pixelorama doesn't match the website oramainteractive.com.
To get the verification through the website .well-known file it will be probably required to change the ID of the app.
https://docs.flathub.org/docs/for-app-authors/verification
